packages/design-system/src/components/contextMenu/components/button.js

```diff
@@ -84,6 +85,9 @@ const Button = forwardRef(function Button(
     onFocus,
     forcePadding = false,
     dismissOnClick = true,
+    children,
+    title,
```

Comment:
💭  since this is related to the tooltip and not the button, I think updating this to `tooltipTitle` would be more descriptive.
```suggestion
    tooltipTitle,
```